code/src/components/Album.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './Album.css';
+import { Icons } from './Icons'
+
+export const Album = props => {
+  const { image, url, name } = props;
+  return (
+    <div className="poster">
+      <div className="album-cover">
+        </div>  
+      
+    <div className="release">
```

Comment:
Keep an eye on your indentation here. It makes it really hard to see if there's an error in your html with an unclosed element or not.